src/etc/git-subtree.sh

```diff
@@ -0,0 +1,1024 @@
+#!/bin/sh
+#
+# git-subtree.sh: split/join git repositories in subdirectories of this one
+#
+# Copyright (C) 2009 Avery Pennarun <apenwarr@gmail.com>
```

Comment:
What is the license of this file? Does it need to be added to https://github.com/rust-lang/rust/blob/master/.reuse/dep5 ?

Reply:
it's the original git license afaict.